geometry/convex-hull-list/src/main/java/com/esri/samples/convex_hull_list/ConvexHullListSample.java

```diff
@@ -67,11 +70,20 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with a basemap and add it to the map view
-      ArcGISMap map = new ArcGISMap(Basemap.Type.TOPOGRAPHIC, 30.0, 0.0, 2);
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+      // create a map view and set its map
       mapView = new MapView();
       mapView.setMap(map);
 
+      // set a viewpoint on the map view
+      mapView.setViewpoint(new Viewpoint(30.0, 0.0, 600000000));
```

Comment:
this is a different scale value than was used in a previous PR https://github.com/Esri/arcgis-runtime-samples-java/pull/599/files#diff-8f2b646da59e98410b35da25b663995af9c16ab65fb85d1d6bea55aeff5de1df

We should try and be consistent with the values that we are swapping: the other values have been consistent between PRs for now :) 

if you haven't already seen it, you can refer to this to get an idea of what the exact replacement values would be, but rounding it down as you have been doing should be fine for samples. https://services.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer

where e.g. an old Level of Detail 12 would now be a new scale of 144447.638572

Reply:
For reference, I've picked up on this point across the changes samples and will correct the scale vs level of detail as per the above link.